rust/lib/lib.rs

```diff
@@ -1,3 +1,5 @@
 pub mod vm;
 pub mod xenctrl;
 pub mod xenstore;
+
+mod bindings;
```

Comment:
Is it possible to have 2 namespaces?
`bindings::xenctrl` & `bindings::xenstore`

For readability purpose.

Reply:
It's possible. But each function in `xenctrl` and `xenstore` has a prefix like`xs_` or `xc`. It's not very useful to add a namespace.